### PR TITLE
Add total person comp cost to the crash details page

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18683

## Testing
**URL to test:** 
Local data model

**Steps to test:**
1. See the new " Total Est. Comprehensive Cost" in the crash details page and the renamed "Maximum Est. Comprehensive Cost"

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vzd/metadata/databases/default/tables/public_crash_injury_metrics_view.yaml [hunk 1]
```diff
@@ -7,17 +7,21 @@ select_permissions:
       columns:
         - bicycle_fatality_count
         - bicycle_sus_serious_injry_count
+        - crash_injry_sev_id
+        - cris_crash_id
         - cris_fatality_count
+        - est_comp_cost_crash_based
+        - est_total_person_comp_cost
```
Comment (roseeichelmann) [inline atd-vzd/metadata/databases/default/tables/public_crash_injury_metrics_view.yaml:14]: just added this and other stuff got rearranged

File: atd-vzd/metadata/databases/default/tables/tables.yaml [hunk 2]
```diff
@@ -41,6 +41,7 @@
 - "!include lookups_veh_mod.yaml"
 - "!include lookups_wthr_cond.yaml"
 - "!include public__column_metadata.yaml"
+- "!include public__cris_import_log.yaml"
```
Comment (roseeichelmann) [inline atd-vzd/metadata/databases/default/tables/tables.yaml:44]: idk why this moved :shrug:

File: atd-vze/src/views/Crashes/crashDataMap.js
Comment (mddilley) [inline atd-vze/src/views/Crashes/crashDataMap.js:1]: Took me a second to understand the difference between these two fields but the DB showed me the way ✨

**Review by mddilley — APPROVED** [review]
Tested locally, and the new field looks great in the UI. The values that I see in the crash page match up with what I'm seeing in `crash_injury_metrics_view` too. 🚢 😎

File: atd-vze/src/views/Crashes/crashDataMap.js [hunk 3]
```diff
@@ -27,7 +27,13 @@ export const createCrashDataMap = isTempRecord => {
...
         },
         est_comp_cost_crash_based: {
-          label: "Est. Comprehensive Cost",
+          label: "Maximum Est. Comprehensive Cost",
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
+          format: "dollars",
+        },
+        est_total_person_comp_cost: {
+          label: " Total Est. Comprehensive Cost",
```
Comment (johnclary) [inline atd-vze/src/views/Crashes/crashDataMap.js:36]: Looks like his leading whitespace space can be removed

**Review by johnclary — CHANGES_REQUESTED** [review]
Looks great! just the smallest possible change needed

**Review by johnclary — APPROVED** [review]
i fixed the whitespace and am gonna merge! 🙏

**Pull request merged**